pkg/services/ngalert/notifier/channels/template_data.go

```diff
@@ -89,6 +89,34 @@ func extendAlert(alert template.Alert, externalURL string, logger log.Logger) *E
 			u.RawQuery = "viewPanel=" + panelId
 			extended.PanelURL = u.String()
 		}
+
+		generatorUrl, gErr := url.Parse(extended.GeneratorURL)
```

Comment:
Can you please add a test that validates the orgID is appended when these are set, but the url is otherwise the same?

Reply:
Hi,

Currently there are two tests covering these scenarios

[// Firing with dashboard and panel ID](https://github.com/grafana/grafana/blob/8911d64574bda76a24b5345d18cb2ba69a483abe/pkg/services/ngalert/notifier/channels/default_template_test.go#L19) - This has the orgId in the annotations and expects the query param to be appended to the generator URL

[// Firing without dashboard and panel ID](https://github.com/grafana/grafana/blob/8911d64574bda76a24b5345d18cb2ba69a483abe/pkg/services/ngalert/notifier/channels/default_template_test.go#L29) - This expects the generated URL without the orgId

Should the above mentioned tests sufice?